docs/guidance/kuberay-with-MCAD.md

```diff
@@ -5,7 +5,7 @@ The multi-cluster-app-dispatcher is a Kubernetes controller providing mechanisms
 ## Use case
 
 MCAD allows you to deploy Ray cluster with a guarantee that sufficient resources are available in the cluster prior to actual pod creation in the Kubernetes cluster. It supports features such as:
-   
+
 - Integrates with upstream Kubernetes scheduling stack for features such co-scheduling, Packing on GPU dimension etc.
 - Ability to wrap any Kubernetes objects.
 - Increases control plane stability by JIT (Just-in Time) object creation.
```

Comment:
Out of curiosity, would you mind explaining this feature?

Reply:
@asm582 may explain this JIT better.
My understanding is mcad creating object only when there is enough resource. In other words, there will not be any pending pods so the control plane stability is improved.

Reply:
My other thought would be, Mcad when paired the InstaScale operator can auto scale out enough k8s worker nodes to run a job and scales down afterwards; This kinda fits the Just in time concept of allocating right amount resources on the assembly line to complete a task; in order to reduce inventory and cost.

Reply:
Yep that's about right! MCAD will not create the underlying Ray resources until there are enough resources to schedule the pod. If InstaScale is enabled, InstaScale will scale up new nodes on your cluster -> MCAD will schedule Ray resources onto new nodes -> Ray Cluster runes -> Once Ray Cluster context is finished and appwrapper is deleted, InstaScale deletes nodes. At no point will there be pending pods/services/routes etc. on your cluster